### PR TITLE
fix issue #11928 suggestions table width

### PR DESCRIPTION
**Pull request by JohnRDOrazio**
BEFORE the fix:
Weblate screenshot BEFORE table fix

AFTER the fix:
Weblate screenshot AFTER table fix

### CODE REVIEW

**Comment by JohnRDOrazio** [comment]
fixes issue https://github.com/WeblateOrg/weblate/issues/11928

**Comment by JohnRDOrazio** [comment]
new screenshot: 
image

File: weblate/static/editor/full.js [hunk 1]
```diff
@@ -578,7 +578,7 @@
           .attr("dir", this.state.dir)
           .html(el.html),
       );
-      row.append($("<td>").html(el.diff));
+      row.append($("<td>").attr("class", "machinery-diff").html(el.diff));
```
Comment (JohnRDOrazio) [inline weblate/static/editor/full.js:581]: another approach could be to simply add the already defined `machinery-text` class which uses the rule `word-wrap: anywhere`, and has pretty much the same effect

**Comment by JohnRDOrazio** [comment]
this PR force-overwrites PR #11929 (which was therefore automatically closed)

File: weblate/static/editor/full.js (hunk 1 above)
Comment (nijel) [reply weblate/static/editor/full.js:581]: That sounds even better. There is no reason to apply different logic to diff than to the full text.

Comment (nijel) [inline weblate/static/editor/full.js:581]:
```suggestion
      row.append($("<td>").attr("class", "machinery-text").html(el.diff));
```

File: weblate/static/style-bootstrap.css [hunk 2]
```diff
@@ -1621,6 +1621,10 @@ input.color_edit:checked + label {
   word-wrap: anywhere;
 }
 
+.machinery-diff {
+  word-break: break-word;
+}
+
```
Comment (nijel) [inline weblate/static/style-bootstrap.css:1627]:
```suggestion
```

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**